rfc/0060-mod-menu.md

```diff
@@ -0,0 +1,73 @@
+
+## Summary
+
+This document outlines the process for creating an official Mod Menu for usage with Quilt alongside an accompanying team that would maintain it. This mod menu will act as a way to view loaded mods, change configs, read credits, and get links to contacts from within Minecraft.
+
+
+## Motivation
+   As of right now, we do not supply an official Mod Menu for Quilt. The current solution is TerraformerMC's Mod Menu for Fabric, and that does not fully support our additions (Full Quilt support, Quilt Config, quilt.mod.json spec improvements, etc.). Having a standardized Mod Menu would allow for support for these features. Furthermore, it would also be an officially endorsed mod, which would help us lead players to it, and we at The Quilt Project would be able to ensure its quality.
+
+
+## Explanation
+
+### Why are we doing this?
+Native Quilt Support, which lets us add additional features from things like quilt.mod.json or Quilt Config without issue.
+Standardization, allowing an official solution compared to Fabric's unofficial Mod Menu
+
+### Why not a fork of Terraformer's Mod Menu?
+The author of the Mod Menu himself, Prospector, said that ModMenu is "janky" - it is likely not worth the effort to clean up code AND integrate with other Quilt Standards. 
+
+### How will we deal with Fabric Mods that require configs?
+We will supply the Mod Menu API to allow for backwards compatibility with configs in Fabric mods.
+
+
+## Drawbacks
+
+### Potentially Unnecessary
+   It maybe be considered to be unnecessary to make this as the current Mod Menu solution on Fabric works fine. It also may complicate the user experience to have multiple mod menu mods. 
+   
+Fabric's Mod Menu is a separate mod, however, so it can simply be removed if a user finds it distracting.
+
+### Removes User Choice
+
+Some people may not want to be forced upon a Mod Menu in that case we are taking away their choice on whether or not to have a Mod Menu. 
+    
+This, however, is almost a non-issue; mod menus are often lightweight, and they are often the best way to abstract access to a mod's config. 
```

Comment:
I am not really clear on how the implementation of this would work, but I would suggest that it still be optional? For example, if I create a highly customized and integrated modpack, I would not want there to be a mod menu (and users should not need to access configs anyway).

Reply:
We can prob integreate a config option for removing the button from the main menu.